format.sh

```diff
@@ -20,7 +20,7 @@ fi
 
 echo "Formatting C files. This will take a bit"
 echo "Running clang-format..."
-clang-format ${FORMAT_OPTS} src/**/*.c
+clang-format-11 ${FORMAT_OPTS} src/**/*.c
```

Comment:
Should we also format the `.h` files?

Reply:
I don't think OOT does. In this case, I want to do the same thing.